app/src/main/java/org/oppia/android/app/profileprogress/ProfileProgressHeaderViewModel.kt

```diff
@@ -42,7 +43,9 @@ class ProfileProgressHeaderViewModel(activity: AppCompatActivity, fragment: Frag
   }
 
   fun clickOnViewAll() {
-    routeToRecentlyPlayedActivity.routeToRecentlyPlayed()
+    routeToRecentlyPlayedActivity.routeToRecentlyPlayed(
+      activity.getString(R.string.recently_played_activity)
```

Comment:
Similar to `getHeader()` function from `PromotedStoryListViewModel.kt` you should use `resourceHandler` to fetch strings here and everywhere else so that you can get translated strings.